src/main/java/com/theelixrlabs/healthcare/response/SuccessResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.theelixrlabs.healthcare.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Represents a generic success response structure for API responses.
+ *
+ * @param <User> the type of data included in the response.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SuccessResponse<User> {
+    /**
+     * Indicates whether the operation was successful.
+     */
+    private Boolean success;
```

Comment:
Any specific reason to use Boolean instead boolean ? 
Making use of primitive datatypes will help in these scenarios